relationships/synthesis/INFRA-NR_DOCKER_CONTAINER-to-EXT-SERVICE.yml

```
@@ -0,0 +1,32 @@
+relationships:
+  - name: nrInfraDockerContainerEXTService
+    version: "1"
+    origins:
+      - OpenTelemetry
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: entity.type
+        anyOf: ["SERVICE"]
+    relationship:
+      expires: P75M
+      relationshipType: HOSTS
+      source:
+        buildGuid:
+          account:
+            attribute: accountId
+          domain:
+            value: INFRA
+          type:
+            value: CONTAINER
+            valueInGuid: NA
+          identifier:
+            fragments:
+                value: "docker:"
+              - attribute: container.id
+            hashAlgorithm: FARM_HASH
+      target:
+        extractGuid:
+          attribute: entity.guid
+          entityType:
+            attribute: entity.type
```

You can set the entity type directly as `SERVICE` using

```suggestion
            value: "SERVICE"
```

and delete the attribute `entity.type` under `conditions`